### PR TITLE
faq.rst: sync between different Syncting folders

### PR DESCRIPTION
**Pull request by novoid**
FAQ: sync between different Syncting folders

Details: https://github.com/syncthing/docs/issues/268

### CODE REVIEW

**Comment by AudriusButkevicius** [comment]
This depends onthe scan order. If you have folder A and B, and move files from A to B, if A gets scanned first, it will announce removal of the files to others who will remove the files. As you rescan B, B will announce addition of new files, and other peers will have nowhere to get them from apart from redownloading them.

If B gets rescanned first, B will announce additions first, remote peers will reconstruct the files (not rename, more like copy block by block) from A, and then as A gets rescanned remove the files from A.

**Comment by ProactiveServices** [comment]
@novoid I recently moved lots of photos and videos from a synced `camera` folder to an `archive` folder. Some files were copied by Syncthing locally but most were deleted and re-downloaded, as described in your issue #268.
As I understand it, for Syncthing to be able to handle this correctly would require some complex work. At the moment the answer is that Syncthing does not specially handle this case, and the user should expect most files to be re-downloaded.

**Comment by AudriusButkevicius** [comment]
A workaround would be to copy first, rescan, wait for others to rebuild on remote ends, and then delete

**Comment by novoid** [comment]
I updated my pull request with an answer.

**Comment by AudriusButkevicius** [comment]
@st-review lgtm

**Comment by st-review** [comment]
@AudriusButkevicius: Noted! Need another LGTM or explicit merge command.

File: users/faq.rst [hunk 1]
```diff
@@ -226,7 +226,21 @@ programs to achieve this such as rsync or Unison.
 When I do have two distinct Syncthing-managed folders on two hosts, how does Syncthing handle moving files between them?
```
Comment (ProactiveServices) [inline users/faq.rst]: A more compact title would be "Does Syncthing handle moving files between shared folders?"

**Comment by calmh** [comment]
@st-review merge it

@ProactiveServices I agree, do you mind just pushing a (non-PR) commit to clear up the title?

**Comment by calmh** [comment]
argh

**Comment by st-review** [comment]
:ok_hand: Merged as 4ad807f54d0443952b854bc473cad8a04abe1d47. Thanks, @novoid!

**Comment by calmh** [comment]
Dat commit message

http://www.reactiongifs.com/wp-content/uploads/2013/03/computer-facepalm.gif

**Pull request closed without merging**